### PR TITLE
EXUI 4.0.5

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
LGTM

**Review by paddybasi — APPROVED** [review]

**Review by sreekanthpuligadda — APPROVED** [review]
looks good.

File: package.json [hunk 1]
```diff
@@ -104,6 +106,9 @@
     "isomorphic-fetch": "^3.0.0",
     "jwt-decode": "^2.2.0",
     "launchdarkly-js-client-sdk": "^2.16.3",
+    "mochawesome": "^6.1.1",
```
Comment (paul-graham) [inline package.json:109]: Shouldn't these _just_ be in `devDependencies`?

**Review by paul-graham — COMMENTED** [review]
Apart from my comment about the `package.json`, I don't see anything that looks squiffy.

**Pull request merged**